scb-scanprocesses/zap-process/src/main/resources/forms/zap/configure-scanner-details.html

```diff
@@ -17,6 +17,7 @@
     	});
     	camForm.on('submit', function () {
     	    $scope.targetList = $scope.targetList.map(function (target) {
+				<!-- target.attributes.ZAP_SCANNER_DELAY_IN_MS = splitOnNewline(target.attributes.ZAP_SCANNER_DELAY_IN_MS); -->
```

Comment:
Either this is an intended comment (to be removed) or an equivalent line for the other property (thread count per host) is missing